import time
import 
from .rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
.configuration.access_token = 'YOUR_ACCESS_TOKEN'

# create an instance of the API class
api_instance = .PatchWorkforcemanagementManagementunitWorkplanRequest()
management_unit_id = 'management_unit_id_example' # str | The ID of the management unit, or 'mine' for the management unit of the logged-in user.
work_plan_id = 'work_plan_id_example' # str | The ID of the work plan to update
body = PureCloudPlatformClientV2.WorkPlan() # WorkPlan | body (optional)
validation_mode = 'validation_mode_example' # str | Allows to update work plan even if validation result is invalid (optional)

try:
    # Update a work plan
    api_response = api_instance.patch_workforcemanagement_managementunit_workplan(management_unit_id, work_plan_id, body=body, validation_mode=validation_mode)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling PatchWorkforcemanagementManagementunitWorkplanRequest->patch_workforcemanagement_managementunit_workplan: %s\n" % e)